aws/table_aws_codedeploy_deployment_group.go

```diff
@@ -0,0 +1,367 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/codedeploy"
+	"github.com/aws/aws-sdk-go-v2/service/codedeploy/types"
+
+	codedeployv1 "github.com/aws/aws-sdk-go/service/codedeploy"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCodeDeployDeploymentGroup(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_codedeploy_deployment_group",
+		Description: "AWS CodeDeploy Deployment Group",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"deployment_group_name", "application_name"}),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ApplicationDoesNotExistException", "DeploymentGroupDoesNotExistException"}),
+			},
+			Hydrate: getCodedeployDeploymentGroup,
```

Comment:
Ensure camel casing in all places
```suggestion
			Hydrate: getCodeDeployDeploymentGroup,
```